overlord/hookstate/ctlcmd/is_connected.go

```diff
@@ -47,6 +54,12 @@ $ echo $?
 
 Snaps can only query their own plugs and slots - snap name is implicit and
 implied by the snapctl execution context.
+
+The --pid and --aparmor-label options can be used to determine whether
+a plug or slot is connected to the snap identified by the given
+process ID or AppArmor label.  In this mode, additional failure exit
+codes may be returned: 10 if the other process is not snap confined,
+or 11 if the other snap is not connected but uses classic confinement.
```

Comment:
was there are a reason to pick 10 and 11, or it's mostly arbitrary?

Reply:
It was an arbitrary choice: I just wanted to put some distance between them and the 0/1 exit statuses.  There is no code that depends on this interface yet, so we can easily change them to something else if you'd prefer.

Reply:
ok, it's kind of a nitpick but I would swap their use:

* 10 for a classic confined snap
* 11 if the process is not confined